snapcraft/__init__.py

```diff
@@ -115,7 +115,11 @@
 import os
 import shutil
 
+from snapcraft._help import topic_help  # noqa
+from snapcraft._store import login, logout, upload  # noqa
```

Comment:
Those are imported but not used, so I'd like to call them out explicitly.
It's defacto done like this for a package `__init__.py`